implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -128,34 +128,30 @@ pub(crate) fn multiaddr_to_route(ma: &MultiAddr) -> Option<Route> {
 }
 
 pub fn setup_logging(verbose: u8) {
-    let ockam_crates = [
-        "ockam",
-        "ockam_node",
-        "ockam_core",
-        "ockam_command",
-        "ockam_identity",
-        "ockam_channel",
-        "ockam_transport_tcp",
-        "ockam_vault",
-        "ockam_vault_sync_core",
-    ];
```

Comment:
I think we need to keep this list because otherwise logging will be also applied to dependency crates which we really don't care about (mostly `tokio` and `mio` are common culprits for spamming debug and trace lines